packages/paste-website/src/components/tokens-list/token-card/token-example/TextColorExample.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {Text} from '@twilio-paste/text';
+import type {ThemeShape} from '@twilio-paste/theme';
+import type {Properties} from 'csstype';
+import {TextExample} from './TextExample';
+import {contrastScore} from '../../../../utils/contrastScore';
+
+export const TextColorExample: React.FC<{
+  value: keyof ThemeShape['textColors'];
+  backgroundColor: Properties['backgroundColor'];
+}> = ({value, backgroundColor}) => {
+  const contrastRating = contrastScore(value, backgroundColor);
+
+  return (
+    <Box display="flex" flexDirection="column" alignItems="center">
+      <TextExample color={value} />
+      <ScreenReaderOnly>Accessibility rating: </ScreenReaderOnly>
+      <Text as="p" fontSize="fontSize10" lineHeight="lineHeight10" marginTop="space20" marginBottom="space0">
```

Comment:
Suggestion: super small nit, but i would move the ScreenReaderOnly text into the Text. This is because SRO defaults to a span which has no semantic meaning, but the Text is a paragraph.